src/dataStore/DripsDataProxy.sol

```diff
@@ -0,0 +1,168 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.8.20;
+
+import {DripsDataStore} from "./DripsDataStore.sol";
+import {Drips, StreamReceiver, IERC20, SplitsReceiver} from "../Drips.sol";
+import {Managed} from "../Managed.sol";
+
+/// @notice A data proxy for `Drips`.
+/// Large parameters aren't passed in calldata, but instead their hashes are accepted,
+/// and the actual data is loaded from `DripsDataStore`.
+/// The data must be explicitly stored in `DripsDataStore` before its usage.
+///
+/// In some cases using the proxy is easier and cheaper than calling the driver directly.
+/// For example multisigs and governance contracts often need the whole calldata
+/// provided as a binary blob, which then sometimes needs to be stored on-chain.
+/// It's easier to build calldata consisting of a flat list of scalar parameters,
+/// and calldata is much smaller when large lists are substituted with their hashes.
+contract DripsDataProxy is Managed {
```

Comment:
Why making it `Managed` it is a contract with no state or `storage`?

Reply:
`Managed` is mostly about adding upgrades and pausing to the contract. The storage helpers is a bonus for upgrading, but it doesn't need to be used, the compiler will probably just remove its logic from the bytecode.